bears/c_languages/AStyleLintBear.py

```diff
@@ -0,0 +1,56 @@
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.bearlib.spacing.SpacingHelper import SpacingHelper
+from dependency_management.requirements.DistributionRequirement import (
+    DistributionRequirement)
+
+
+@linter(executable='astyle',
+        output_format='corrected',
+        use_stdin=True)
+class AStyleLintBear:
+    """
+    Artistic Style is a source code indenter, formatter,
+    and beautifier for the C, C++, C++/CLI, Objective‑C,
+    C# and Java programming languages.
+
+    For more information, consult <http://astyle.sourceforge.net/astyle.html>.
+    """
+
+    LANGUAGES = {'C', 'C++', 'Objective-C', 'C#', 'Java'}
+    REQUIREMENTS = {DistributionRequirement(apt_get='astyle')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_FIX = {'Formatting'}
+
+    @staticmethod
+    def create_arguments(filename, file, config_file,
+                         astyle_bracket_style: str='',
```

Comment:
I don't think that is right. If I set the default to allman it means the linter will always be run with command line option --style=allman. Users may not want that. Command line options take precedence over config file options. So If a user had something like --stye=java inside his config file, it will not work because the linter is always run with --style=allman which takes precedence over the config file.